docs/UserGuide.md

```diff
@@ -313,26 +314,31 @@ Format: `clear`
 
 ### Importing data : `import`
 
-Imports a .csv file compatible with EmployeeManager into the application. It will overwrite the existing
-data stored in the application.
+Imports an EmployeeManager data file into the application.
```

Comment:
Would it be clearer to state the file types accepted?

Reply:
Good idea, I have added that line